lib/core/src/sdk.rs

```diff
@@ -345,6 +337,12 @@ impl LiquidSdkBuilder {
         });
         Ok(sdk)
     }
+
+    pub async fn connect(&self) -> Result<Arc<LiquidSdk>> {
+        let sdk = self.build()?;
+        sdk.start().await?;
```

Comment:
What is the reason for moving the start here? I prefer to keep the LiquidSdkBuilder as a builder for sdk object.

Reply:
Fixed in [2971128](https://github.com/breez/breez-sdk-liquid/pull/792/commits/2971128c0eb7673c11ff8b322a8b9e2e130567ab)